### PR TITLE
Cleanup expired EstEID card ATR-s

### PR DESCRIPTION
**Pull request by metsma**
Signed-off-by: Raul Metsma <raul@metsma.ee>

##### Checklist
- [ ] Documentation is added or updated
- [ ] New files have a LGPL 2.1 license statement
- [X] Tested with the following card: EstEID
	- [X] tested PKCS#11
	- [ ] tested Windows Minidriver
	- [ ] tested macOS Tokend

### CODE REVIEW

File: etc/opensc.conf.in [hunk 1]
```diff
@@ -379,33 +379,11 @@ app default {
 		# driver = "piv";
 	# }
 
-	# Estonian ID card and Micardo driver sometimes only play together with T=0
+	# Micardo driver sometimes only play together with T=0
 	# In theory only the 'cold' ATR should be specified, as T=0 will
 	# be the preferred protocol once you boot it up with T=0, but be
 	# paranoid.
 	#
```
Comment (frankmorgner) [inline etc/opensc.conf.in]: I'm puzzled: Why do you still need this comment if the configuration below is deleted?

Comment (metsma) [reply etc/opensc.conf.in]: Look below is D-Trust cards also Micardo cards

Comment (frankmorgner) [reply etc/opensc.conf.in]: thanks!

**Comment by metsma** [comment]
I don't have D-Trust cards and they are not in use in Estonia

**Comment by frankmorgner** [comment]
Sorry, my question was about EstEID, does it support encryption?

**Comment by metsma** [comment]
Yes, but I dont see how it is related?
Micardo cards are all expired and obosolete in estonia

**Comment by frankmorgner** [comment]
Imagine, you've received an encrypted mail back in 2015. You may want to be able to decrypt this with your old card even if this type of card is not manufactured or valid anymore.

**Comment by metsma** [comment]
Then you can use previous opensc version?
And coverment stopped issuing micardo cards 2011 or earlier

**Comment by frankmorgner** [comment]
You cannot easily use two versions of OpenSC at the same time (e.g. for switching between old and new cards). Anyway, the change is fine with me if you don't see any conflict.

**Comment by metsma** [comment]
These cards are old and if somebody has still encrypted documents, they can re encrypt to new cards with previous version.

**Pull request merged**